packages/wdio-spec-reporter/src/index.ts

```diff
@@ -126,6 +128,14 @@ export default class SpecReporter extends WDIOReporter {
         this.printReport(runner)
     }
 
+    /**
+     *  Return text based on color option
+     */
+
+    private _setTextColor(state: string, text: string){
```

Comment:
Let's create a custom chalk instance per reporter, e.g.
```ts
this.chalk = new Chalk(!options.color ? { level: 0 } : {})
```
instead of having this method.

Reply:
It does not allow me to do that. It says:

- TypeError: Chalk is not a constructor

Reply:
please see [Chalk](https://www.npmjs.com/package/chalk#api) docs, the import has to be: `import { Chalk } from 'chalk'`

Reply:
@christian-bromann It kinda works in node but not in code. I didn't actually understand the reason.

This is what I have been using:

```
import { Chalk } from 'chalk'

class SpecReporter extends WDIOReporter {
  private _chalk: ChalkInstance
  this._chalk = new Chalk(!options.color ? { level: 0 } : {})
  }
```
  
It is not working. I get the error;


 ```
FAIL  packages/wdio-spec-reporter/tests/index.test.ts [ packages/wdio-spec-reporter/tests/index.test.ts ]

TypeError: Chalk is not a constructor

 ❯ new SpecReporter packages/wdio-spec-reporter/src/index.ts:63:23
     61|         this._realtimeReporting = options.realtimeReporting || false
     62|         this._showPreface = options.showPreface !== false
     63|         this._chalk = new Chalk(!options.color ? { level: 0 } : {})
       |                       ^
     64|         this._sauceLabsSharableLinks = 'sauceLabsSharableLinks' in options
     65|             ? options.sauceLabsSharableLinks as boolean

 ❯ packages/wdio-spec-reporter/tests/index.test.ts:20:18

```

I have tried using simply their code;

```
import {Chalk} from 'chalk';
class SpecReporter extends WDIOReporter {
  private _chalk: ChalkInstance
  this._chalk = new Chalk({level: 0})
  }
```

 Error still remains same.

```
TypeError: Chalk is not a constructor

 ❯ new SpecReporter packages/wdio-spec-reporter/src/index.ts:63:23

     61|         this._realtimeReporting = options.realtimeReporting || false
     62|         this._showPreface = options.showPreface !== false
     63|         this._chalk = new Chalk({ level: 0 })
       |                       ^
     64|         this._sauceLabsSharableLinks = 'sauceLabsSharableLinks' in options
     65|             ? options.sauceLabsSharableLinks as boolean

 ❯ packages/wdio-spec-reporter/tests/index.test.ts:20:18
```

Reply:
I have no issues creating a chalk instance: here is the patch: 
[chalk.patch](https://github.com/webdriverio/webdriverio/files/14067368/chalk.patch)

Reply:
resolved